src/style.css

```diff
@@ -53,12 +72,14 @@ body {
   border: 2px solid black;
   padding: 5px;
   cursor: pointer;
-  transition: all 1s ease-in-out;
+  /* stylelint-disable-next-line csstree/validator */
```

Comment:
- [ ]  linter is used to flag programming errors, bugs, stylistic errors, and suspicious constructs, so to keep your code readable and understandable by everyone, kindly remove the linters disabling command and please fix all the linters error 🙏